### ISSUE TITLE
import() does not work

### ISSUE THREAD

**Issue by philipahlberg** [webpack-contrib/karma-webpack#316]
**I'm submitting a bug report**

**Webpack version:**
4.1.1

**Webpack Karma version**:
2.0.13

**Karma version**:
2.0.0

**Please tell us about your environment:**
Windows 10

**Browser:** [Chrome 64] (probably others as well)

**Current behavior:**
`import()` fails with the error:
`Error: Loading chunk 0 failed.`

**Expected/desired behavior:**
`import()` should import a script and the exports should be available.

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem along with a gist/jsbin of your webpack configuration.**
See https://github.com/philipahlberg/karma-webpack-dynamic-import for a setup that showcases the problem.
The minimum necessary steps to reproduce is:
1. 
```console
npm install --save-dev karma webpack karma-webpack
```
2. setup a test that uses `import()`
3. execute the test

* **What is the expected behavior?**
`import('./import.js')` should import the file './import.js' and return its exports.

* **What is the motivation / use case for changing the behavior?**
To allow testing things with dynamic imports.

**Comment by jamesopti** [comment]
Im seeing this issue as well but when using Webpack4's `optimization.splitChunks` to split out my vendor bundle.

**Webpack version:**
4.1.1

**Webpack Karma version:**
3.0.0

**Karma version:**
1.7.0

**Please tell us about your environment:**
Mac OSX Sierra 10.12.6

**Browser:** [Chrome 57]

**Current behavior:**
Vendor chunk fails to load

**Expected/desired behavior:**
vendor chunk should be available at `http://localhost:9876/_karma_webpack_/vendor.js`

**Webpack Config:**
```
optimization: {
  splitChunks: {
      cacheGroups: {
        vendor: {
          test: /\/node_modules\//,
          name: 'vendor',
          chunks: 'initial',
          filename: '[name].js',
        },
      },
    },
}
```

**Comment by philipahlberg** [comment]
FWIW, it's also broken for karma-webpack 4.0.0-beta.0 + webpack 4.1.1.

**Comment by jamesopti** [comment]
+1 - This seems to be a regression that's happened multiple times now in a world where code splitting and dynamic imports are only getting more prevalent.

**Comment by ggriffithsIDBS** [comment]
+1 seeing this in latest version with Webpack 3
`Error: Loading chunk 0 failed`

**Comment by tingc10** [comment]
Not sure if this is the same error, but I solved this by setting webpack's `output.chunkFilename` and `output.filename` to `[name].js`. I noticed the chunk was being named to `1-undefined.js`.

**Comment by matthieu-foucault** [comment]
The dynamic import bug on Windows (also reported in #362 ) was fixed in #373. installing `karma-webpack@next` would fix it
